src/app.ts

```diff
@@ -2,11 +2,13 @@ import { FastifyInstance } from 'fastify';
 import getMessage from "./getMessage";
 import getCharger from './getCharger';
 import getAllCharger from './getAllCharger';
+import getHistoryCharger from './getHistoryCharger';
 
 async function app(fastify: FastifyInstance, opts: any){
   fastify.get('/', getMessage);
   fastify.get('/charger', getAllCharger);
   fastify.get('/charger/:id', getCharger);
+  fastify.get('/history_charger', getHistoryCharger);
```

Comment:
¿Qué te parece si le llamamos charges/:id?

Reply:
https://dev.to/dennysjmarquez/todo-lo-que-necesitas-saber-sobre-api-rest-glosario-de-terminos-esenciales-y-mas-29pc